exercises/custom-converter/practice/src/workflows.ts

```diff
@@ -0,0 +1,46 @@
+import type * as activities from './activities';
+import { proxyActivities, log } from '@temporalio/workflow';
+import { User, Result } from './shared';
+
+const { converterActivity } = proxyActivities<typeof activities>({
+  startToCloseTimeout: '10 seconds',
+});
+
+// // Workflow is a standard workflow definition.
```

Comment:
Maybe I'm missing something, but why do you have this commented block here? Looks like this be left over from a previous iteration...